contractsSol5/KyberStorage.sol

```diff
@@ -214,6 +216,27 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers {
         feeAccountedPerType = feeAccountedData;
     }
 
+    function setEntitledRebatePerReserveType(
+        bool fpr,
+        bool apr,
+        bool bridge,
+        bool utility,
+        bool custom,
+        bool orderbook
+    ) external {
+        onlyAdmin();
+        uint256 entitledRebateData;
+
+        if (apr) entitledRebateData |= 1 << uint256(ReserveType.APR);
+        if (fpr) entitledRebateData |= 1 << uint256(ReserveType.FPR);
+        if (bridge) entitledRebateData |= 1 << uint256(ReserveType.BRIDGE);
+        if (utility) entitledRebateData |= 1 << uint256(ReserveType.UTILITY);
+        if (custom) entitledRebateData |= 1 << uint256(ReserveType.CUSTOM);
+        if (orderbook) entitledRebateData |= 1 << uint256(ReserveType.ORDERBOOK);
+
+        entitledRebatePerType = entitledRebateData;
```

Comment:
we can consider that this should always be updated after isFeePaying is updated.
and then must be aligned.
can't have rebate if not feeAccounted.

Reply:
see updated code

Reply:
Yes
Great